src/main/java/org/apache/maven/plugins/dependency/tree/TreeMojo.java

```diff
@@ -479,9 +479,12 @@ public static boolean containsVersion( VersionRange allowedRange, ArtifactVersio
                     return true;
                 }
             }
+            return false;
+        }
+        else
+        {
+            // only singular versions ever have a recommendedVersion
+            return recommendedVersion.compareTo(theVersion) <= 0;
```

Comment:
Spaces missing

Reply:
Corrected. :cry: I don't know how it happened as it is clearly reported by `verify`.